### PR TITLE
Adds mapping and social media icons to the documentation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We recommend using [`yarn`](https://yarnpkg.com/en/). Run `yarn` to install depe
 
 Once the repository has been cloned, you can build the components with:
 
-`npx gulp build`
+`npm gulp build`
```
Comment (fionawhim) [inline README.md]: Nope, should be `npx`.

**Review by fionawhim — COMMENTED** [review]
If that map keeps tripping Percy we can ask Tim to add a fuzz factor to our diffs.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Oops. Didn't mean to commit that. Caught on after a bit.

**Comment by fionawhim** [comment]
lgtm

**Pull request merged**